### DISCUSSION TITLE
how many customer interviews?

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#168] [people; relating]
existing scholars
- https://youtu.be/fqOHNmlMzJY?si=lbk10V4BX7UjUEcU

question: larger potential for lift = smaller test
![image](https://github.com/Data4DM/BayesSD/assets/30194633/cb9f5e06-21d4-4709-8b46-95571020e312)

![image](https://github.com/Data4DM/BayesSD/assets/30194633/1d794742-e083-4d97-af13-8c3633df81b5)

![image](https://github.com/Data4DM/BayesSD/assets/30194633/18d70f2b-3936-4618-aaf3-afb0d21a58ed)

**Comment by hyunjimoon** [comment]
curated list:
<img width="1014" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/a80aa5e0-ba90-464f-b5b8-afe1a09f5d47">
[curated_literature.pdf](https://github.com/Data4DM/BayesSD/files/12848450/curated_literature.pdf)

**Comment by tomfid** [comment]
Larger sample > larger test seems counterintuitive, since sample variation depends on sample size, not population size.

**Comment by tomfid** [comment]
This reminds me of the old "secretary problem" in optimal stopping.
